check_image_building/check_image_building_epouta_production.json

```diff
@@ -0,0 +1,18 @@
+{
+    "base_path" : "/home/imgbuild/log",
+    "environments" : [
+        {
+            "name" : "e",
+            "images" : [
+                "ubuntu-18.04",
+                "ubuntu-20.04",
+                "ubuntu-22.04",
+                "ubuntu-24.04",
+                "centos8_stream",
```

Comment:
CentOS 8 stream is also deprecated, and in our customer communcation we said we'd remove it, so I think we should remove it at the same time from imagebuidler.